test/connectedAccounts.routes.test.js

```diff
@@ -42,7 +42,7 @@ describe('connectedAccounts.routes.test.js: GIVEN an application and group', ()
             expect(err).not.to.exist;
             const baseUrl = 'https://github.com/login/oauth/authorize';
             const apiKeyEnc = '.*';
-            const redirectUri = encodeURIComponent(`${config.host.api}/connected-accounts/github/callback?api_key_enc=${apiKeyEnc}&utm_source=mm&slug=`);
+            const redirectUri = encodeURIComponent(`http://127.0.0.1:.*/connected-accounts/github/callback?api_key_enc=${apiKeyEnc}&utm_source=mm&slug=`);
```

Comment:
That's what allows the test to pass... The passport code seems to resolve to that in the unit tests. The port changes with every run.